apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/components/GroupInfoModal.tsx

```diff
@@ -0,0 +1,203 @@
+              closedSubmissionsOnly,
+              numOpenInstances,
+            },
+            {
+              onSuccess: onHide,
+            },
+          );
+          break;
+        }
+        case 'ungrouped': {
+          mutation.mutate(
+            {
+              action: 'ai_instance_question_group_assessment_ungrouped',
+              closedSubmissionsOnly,
+              numOpenInstances,
+            },
+            {
+              onSuccess: onHide,
+            },
+          );
+          break;
+        }
+        default:
+          assertNever(modalState);
+      }
+    },
+    [modalState, closedSubmissionsOnly, numOpenInstances, mutation, onHide],
+  );
+
+  if (!modalState) return null;
+
+  const getTitle = () => {
+    if (modalState.type === 'all') {
+      return 'Group all submissions';
+    } else if (modalState.type === 'ungrouped') {
+      return 'Group ungrouped submissions';
+    } else {
+      return 'Group selected submissions';
+    }
+  };
+
+  return (
+    <Modal show={true} size="lg" backdrop="static" keyboard={false} onHide={handleClose}>
+      <form onSubmit={handleSubmit}>
+        <Modal.Header closeButton>
+          <Modal.Title>{getTitle()}</Modal.Title>
+        </Modal.Header>
+
+        <Modal.Body>
+          <p>
+            Groups student submission answers based on whether they{' '}
+            <strong>match the correct answer exactly.</strong>
+          </p>
+
+          <p>Answers that match go into one group, and those that don't are grouped separately.</p>
+
+          <p>
+            To enable grouping, the correct answer must be provided in <code>pl-answer-panel</code>.
+          </p>
+
+          <p>
+            Grouping checks for exact equivalence to the final answer, considering only the boxed or
+            final answer to form groups.
+          </p>
+
+          <p>Examples of what can and can't be grouped:</p>
+
+          <div
+            class="d-grid border rounded overflow-hidden"
+            style={{ gridTemplateColumns: '1fr 1fr' }}
+          >
+            <div class="px-2 py-1 bg-light fw-bold border-end">Can group</div>
+            <div class="px-2 py-1 bg-light fw-bold">Can't group</div>
+
+            <div class="px-2 py-1 border-top border-end">Mathematical Equations</div>
+            <div class="px-2 py-1 border-top">Essays</div>
+
+            <div class="px-2 py-1 border-top border-end">Mechanical Formulas</div>
+            <div class="px-2 py-1 border-top">Free Response Questions</div>
+
+            <div class="px-2 py-1 border-top border-end">Exact String Inputs</div>
+            <div class="px-2 py-1 border-top">Freeform Code</div>
+
+            <div class="px-2 py-1 border-top border-end">
+              Handwritten submissions with 1 correct answer
+            </div>
+            <div class="px-2 py-1 border-top">Handwritten submissions with 2+ correct answers</div>
+          </div>
+
+          {numOpenInstances > 0 && (
```

Comment:
UX nit: it feels weird to show this when checkboxes are in play. But I suspect this is mimicking what master was already doing, so no action needed here.